jedi_language_server/server.py

```diff
@@ -82,8 +83,11 @@ def bf_initialize(self, params: InitializeParams) -> InitializeResult:
             server.initialization_options = InitializationOptions.parse_obj(
                 params.initialization_options
             )
-        except ValidationError:
-            pass
+        except ValidationError as error:
+            msg = f"Invalid InitializationOptions, using defaults: {error}"
+            server.show_message(msg, msg_type=MessageType.Error)
+            server.show_message_log(msg, msg_type=MessageType.Error)
+            server.initialization_options = InitializationOptions()
```

Comment:
my intention had been that this was handled by creating default options at https://github.com/pappasam/jedi-language-server/blob/0cb835600924dc7d3f7a1c67e810811e588876f6/jedi_language_server/server.py#L142, not sure if this was broken?

logging the error is clearly a good idea!

Reply:
Ah, yes, this is redundant, I'll fix now